### ISSUE TITLE
Provide configuration switch from "smooth" to "instant" scrolling in grid

### ISSUE THREAD

**Issue by mike-seger** [vitmalina/w2ui#2271]
**Describe the problem your feature will solve.**
Currently it does not seem to be possible to switch from "smooth" to "instant" scrolling by configuration when instantiating a grid. The documentation shows no obvious configuration.

**Describe the solution you'd like**
A value to set the scrolling behaviour in the configuration would allow higher performance for people who have large data sets.

**Describe alternatives you've considered**
Alternatively the behaviour could be read from CSS, but that would make it less transparent.

**Additional context**
When comparing to other grids like ag-grid and jspreadsheet, w2ui really shines with its high performance, but the smooth scrolling put the breaks down..

**Comment by vitmalina** [comment]
Ade you talking about `w2ui.grid.scrollIntoView(10050, true)`?

**Comment by mike-seger** [comment]
Actually I mean the smooth scrolling effect, when using PageUp / -Down.
I made a modified version of w2ui grid, where I just hardcoded everything to "instant" scrolling here:

https://mike-seger.github.io/grid-experiments/w2ui/?rows=10000

I like the consistent page jumps, but could not find any other way to configure the "instant" flag throughout. Compared to the other contenders, ag-grid and jspreadsheet, your w2ui is the only option for really large grids.
